src/Our.Umbraco.StackedContent/Web/UI/App_Plugins/StackedContent/js/stackedcontent.controllers.js

```diff
@@ -36,6 +54,36 @@
             $scope.model.value.splice(idx, 1);
         }
 
+        $scope.copyToLocalStorage = function (evt, idx) {
+            var stackedContentItem = Object.assign({}, $scope.model.value[idx]);
+            stackedContentItem.key = "";
```

Comment:
I think we might need to populate the key here rather than just wiping it out. Problem is, this task is usually done in InnerContent which has a generateUid method. At it's simplest, we might need to expose generateUid from InnerContent or more, do we move the copy / paste methods (or some part of them anyway) to the inner content package? @leekelleher any thoughts?

Reply:
I think moving the functionality to InnerContent would be a good choice, (sorry if this makes more work).

Reply:
Okay - I'll do my best to submit a PR for InnerContent too. I haven't worked directly with it yet so if I run into a snag I may end up asking you a question or 2 if that's alright.

Reply:
@markadrake Please do ask any questions. I appreciate the complexities in dealing with two separate projects/repositories.